doc/.gitignore

```diff
@@ -1,11 +1,9 @@
-build
 /html
 
-/builtins
-/modules
-/primers
-/examples
+/rst/builtins
+/rst/modules
+/rst/primers
+/rst/examples
 
-tools/chpldoc/man.rst
-tools/chplvis/examples
-usingchapel/man.rst
+man.rst
```

Comment:
`/man.rst` would refer to `doc/man.rst` -- however, `man.rst` applies to any file named `man.rst` (both the `chpldoc` and `chpl` man pages)